lib/stream.js

```diff
@@ -183,6 +187,10 @@ module.exports = class UDXStream extends streamx.Duplex {
 
     if (drained) cb(null)
     else this._onwrite = cb
+
+    if (buffer.byteLength > this.mtu - 20 /* header */) {
+      this.emit('fragmentation')
```

Comment:
should we only emit this once ever?

Reply:
I guess we should find another event name if we do that. Callers can also always do `stream.once('fragmentation')`.

Reply:
is just emitted a lot, always a sucker for reducing overhead 😬

Reply:
(i remember in streamx the data event was a bottleneck)

Reply:
what about `mtu-exceeded` and then we do it once?

Reply:
All set 👌